entrypoint.go

```diff
@@ -34,6 +35,9 @@ var userConfigFile = "/etc/rungmp/config.yaml"
 var otelConfigFile = "/run/rungmp/otel.yaml"
 var configRefreshInterval = 20 * time.Second
 var selfMetricsPort = 0
+var livenessProbePort = 13133
```

Comment:
I used this because this is the port we set up for the OTel sidecar https://cloud.google.com/run/docs/tutorials/custom-metrics-opentelemetry-sidecar#ship-code

Its the default port for the healthcehck extension in OTel. But maybe we don't want to reuse this port because of that? I don't have strong opinions - wdty @jefferbrecht?

Reply:
As long as there's never a reason to run both at the same time then it should be fine.